src/app/(dashboard)/explorer/page.tsx

```diff
@@ -101,25 +99,25 @@ async function YourGuildsSection() {
           </div>
         }
       >
-        <Suspense fallback={<YourGuildsSkeleton />}>
-          <YourGuilds />
+        <Suspense fallback={<AssociatedGuildsSkeleton />}>
+          <AssociatedGuilds />
         </Suspense>
       </AuthBoundary>
     </section>
   );
 }
 
-async function YourGuilds() {
-  const auth = await getTokenFromCookie();
-  if (!auth) return;
-
-  const { items: myGuilds } = await getAssociatedGuilds({
-    userId: auth.userId,
-  });
+async function AssociatedGuilds() {
+  let associatedGuilds: Schemas["Guild"][];
+  try {
+    associatedGuilds = (await getAssociatedGuilds()).items;
+  } catch {
+    return;
```

Comment:
We could display an error message here later

Reply:
And I'm not sure if we should fetch these guilds this way, or we should just use TanStack here too (so prefetching on server side => passing the data to the query). This way we could still render these server side request time, but we could also update the "your guilds" section when the user creates a new guild, deletes an existing one, or leaves/joins a new one)

Reply:
> We could display a proper error message here

Error now propagates.